Src/Plugins/Security/SEC101_004.FacebookAppCredentialsValidator.cs

```diff
@@ -147,14 +147,16 @@ private ValidationState RetrieveInformation<T>(string url, string id, ref string
                     }
 
                     case System.Net.HttpStatusCode.BadRequest:
+                    case System.Net.HttpStatusCode.InternalServerError:
```

Comment:
>InternalServerError [](http://example.com/codeflow?start=51&length=19)

I see a confusing phenomenon, some ids return bad request, others return internal server error. I don't see anything obvious that would distinguish between these conditions, so we'll merge them.